src/components/GameView/OpponentRequestedStalemateDialog.vue

```diff
@@ -53,9 +58,9 @@ export default {
       get() {
         return this.modelValue;
       },
-      set() {
-        // do nothing - parent controls whether dialog is open
-      },  
+      set(val) {
+        this.$emit('update:modelValue', val);
+      },
```

Comment:
This fixes a bug where the dialog wouldn't close when the request completed. Normally we handle this with socket updates but since requesting a stalemate should generally close the dialog it seemed reasonable to close it here unlike the moves you make through a dialog

Reply:
Removed in favor of handling the update in the store like the other dialogs